sky/unit/test/widget/build_utils.dart

```diff
@@ -114,4 +129,13 @@ class WidgetTester {
     Component.flushBuild();
     RenderObject.flushLayout();
   }
+
+  void pumpPaintFrame(WidgetBuilder builder) {
```

Comment:
Perhaps that's possible. At the moment, doing so causes global_key_test.dart to fail:
```
rendering/object.dart': Failed assertion: line 710 pos 12: '!_needsCompositingBitsUpdate' is not true.

I'll add a TODO for now.